frontend/src/components/User/User.js

```diff
@@ -1,78 +1,21 @@
+import '../../config';
 import React from "react";
 import axios from "axios";
```

Comment:
Coding standards: would it be possible to move the axios import before the react one?